lib/tools/adb-commands.js

```diff
@@ -1632,7 +1632,7 @@ methods.getSetting = async function getSetting (namespace, setting) {
  * operation may take up to several minutes.
  *
  * @param {?number} timeout [120000] - Command timeout in milliseconds
```

Comment:
should we also fix types that start with a question mark (e.g. move it to the end)?

Reply:
yes, but I'm not going to do that right now.  it seems to interpret `?` anywhere in the type as `null`.  either way, we probably should not be using `?` since it means one thing in jsdoc (`null`) and another thing in TS (unset or `undefined`) and is thus confusing.